examples/grpc/grpc-unary-server/main_test.go

```diff
@@ -82,3 +69,12 @@ func TestHelloProtoMethods(t *testing.T) {
 	assert.Equal(t, "Hello World", resp.GetMessage())
 	assert.Equal(t, "message:\"Hello World\"", resp.String())
 }
+
+func TestMain(m *testing.M) {
+	os.Setenv("GOFR_TELEMETRY", "false")
+
+	go main()
+	time.Sleep(300 * time.Millisecond) // wait for server to boot
+
+	os.Exit(m.Run())
```

Comment:
```suggestion
	m.Run()
```

This is the recommended way now. Previously calling os.Exit was necessary in TestMain, it's no longer the case